### PR TITLE
add bm60v2_poker

### PR DESCRIPTION
**Pull request by bdtc123**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by sigprof — CHANGES_REQUESTED** [review]

File: keyboards/bm60v2_poker/config.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#pragma once
```
[the remark this answers is not in the document]
Comment (drashna) [reply keyboards/bm60v2_poker/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/bm60v2_poker/config.h [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+#    define DISABLE_RGB_MATRIX_BAND_SPIRAL_SAT
+#    define DISABLE_RGB_MATRIX_RAINDROPS
+#    define DISABLE_RGB_MATRIX_JELLYBEAN_RAINDROPS
+#    define DRIVER_ADDR_1 0b1010000
+#    define DRIVER_COUNT 1
+#    define DRIVER_1_LED_TOTAL 61
+#    define DRIVER_LED_TOTAL DRIVER_1_LED_TOTAL
+
+#if defined(RGB_MATRIX_ENABLE) && defined(RGBLIGHT_ENABLE)
+    #define RGB_MATRIX_DISABLE_KEYCODES
```
Comment (drashna) [inline keyboards/bm60v2_poker/config.h:74]:
```suggestion
#    define RGB_MATRIX_DISABLE_KEYCODES
```

File: keyboards/bm60v2_poker/rules.mk [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite        # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/bm60v2_poker/rules.mk:10]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by peepeetee** [comment]
Take a look at my #15040, which concerns bm60poker
see if you can target develop and move this to bm60poker/rev2, or wait until next breaking change

**Comment by bdtc123** [comment]
> 我的#15040，它与 bm60poker 相关， 您是否可以将开发目标移至 bm60poker/rev2，或者下次看看重大变化



> 看看我的#15040，它与 bm60poker 相关， 看看您是否可以将开发目标移至 bm60poker/rev2，或者等到下一次重大更改



> Take a look at my #15040, which concerns bm60poker see if you can target develop and move this to bm60poker/rev2, or wait until next breaking change

Ok，may i have your email address?

**Comment by peepeetee** [comment]
> 
> 
> > 我的#15040，它与 bm60poker 相关， 您是否可以将开发目标移至 bm60poker/rev2，或者下次看看重大变化
> 
> > 看看我的#15040，它与 bm60poker 相关， 看看您是否可以将开发目标移至 bm60poker/rev2，或者等到下一次重大更改
> 
> > Take a look at my #15040, which concerns bm60poker see if you can target develop and move this to bm60poker/rev2, or wait until next breaking change
> 
> Ok，may i have your email address?

8行(
看我profile啊

**Comment by peepeetee** [comment]
As I've mentioned in the description of #15103, after a discussion with @bdtc123, we decided to have me take over this PR and reincarnate it as #15103, which is already merged. This PR should be able to be safely closed now.

**Pull request closed without merging**